packages/moonstone/DayPicker/DayPicker.js

```diff
@@ -198,11 +233,37 @@ const DayPickerBase = class extends React.Component {
 		}
 	}
 
+	/**
+	 * Determines whether it should return ariaLabel passed in prop.
+	 *
+	 * @param {Number} selected day type
+	 * @param {String[]} selected day aria labels
+	 * @param {String[]} selected day strings
```

Comment:
it should be `* @param {String[]} Array of long day strings`

Reply:
I agree and will fix.